hack/single-node/user-data.sample

```diff
@@ -38,7 +42,8 @@ coreos:
           --lock-file=/var/run/lock/kubelet.lock \
           --network-plugin=cni \
           --node-labels=node-role.kubernetes.io/master \
-          --pod-manifest-path=/etc/kubernetes/manifests
+          --pod-manifest-path=/etc/kubernetes/manifests \
+          --rotate-certificates
```

Comment:
shouldn't it be `--bootstrap-kubeconfig` and `--require-kubeconfig` as per https://kubernetes.io/docs/admin/kubelet-tls-bootstrapping/#kubelet-configuration ?

Reply:
This is actually intentional. We want to start static pods even if the rotation hasn't completed. `--bootstrap-kubeconfig` will block, but `--rotate-certificates` wont, even though they both have the same effect.